_includes/banner.html

```diff
@@ -18,4 +19,7 @@ <h1 class="text-lg-center">Giving you a voice in the #OpenSource Tech community
             <p><b>WeRockTech</b> is a youtube channel which so far features discussions & conversations with people in tech who have either contributed by way of their achievements or are new to the community and starting explore how they can contribute & get involved, hopefully this will encourage more to join & do similar</p>
         </div>
     </div>
+
+    <center><p><b>WeRockTech</b> is a youtube channel which so far features discussions & conversations with people in tech who have either contributed by way of their achievements or are new to the community and starting explore how they can contribute & get involved, hopefully this will encourage more to join & do similar</p></center>
```

Comment:
This is a duplicate of the text on [line 19](https://github.com/WeRockTech/werocktech.github.io/pull/25/files#diff-c958d06fb7b14ad74087a68dfb718531R19)

I think it looks better next to the video rather than underneath it. 

![db91cd5a-4ff6-11e6-9877-8e8a90abebc6](https://cloud.githubusercontent.com/assets/624760/17053075/27ed13a2-4ff8-11e6-9b72-dc58f647ac1c.png)


What are your thoughts?